rdkit/Chem/Draw/SimilarityMaps.py

```diff
@@ -53,6 +53,10 @@
 from rdkit.Chem import rdDepictor
 from rdkit.Chem import rdMolDescriptors as rdMD
 
+def _CleanFpInfoAttr_(mol):
+  if hasattr(mol, '_fpInfo'):
+    delattr(mol, '_fpInfo')
```

Comment:
should probably be called `_DeleteFpInfoAttr` because it removes it. Cleaning gives the impression it is still there.

I would also move this to the end of the function `GetAtomicWeightsForFingerprint`.

Reply:
OK, I got your idea! I would fix it in the next patch. I would change the name as requested. Thanks you for your review.